src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random randomNumber = new Random();
```

Comment:
```suggestion
        Random random = new Random();
```

Reply:
I don't see any problems. one is for the random number and one is for the random color. Both are implemented in separate classes.